### PR TITLE
Fix #86: bb test runner

### LINKED ISSUE #86: Support usage in Babashka environments
Babashka is a dialect of Clojure that supports the [Small Clojure Interperter](https://github.com/borkdude/sci) sci as a graalvm compiled native image.  It's main usecase is quick startup times, so it's suitable as a bash replacement, but also in other environments such as in AWS lambda deployments.

As an experiment I tried porting integrant to work with babashka, only minimal changes are required.  You can see this work in my [integrant fork](https://github.com/RickMoynihan/integrant), and in my fork of [dependency](https://github.com/rickmoynihan/dependency).

In the process of this work we've identified a few areas of improvement within sci and babashka that will require even less changes to integrant and dependency.  Specifically these two issues, one of which is already fixed in master of babashka:

- [x] https://github.com/borkdude/babashka/issues/581
- [ ] https://github.com/borkdude/sci/issues/419

@weavejester I'm wondering if once these two issues are resolved, whether you'd consider officially incorporating babashka support into integrant?  And whether I should work my forks changes up into a revised PR, that reverts the commits related to those issues and includes running the tests under babashka in the travis CI environment?

### PR DESCRIPTION
**Pull request by borkdude**

### CODE REVIEW

**Comment by borkdude** [comment]
@weavejester I updated this PR with the freshly released babashka version and should be ready for merge. Let me know if you want any changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * **BREAKING CHANGE:** removed old `expand` function
 * Added `assert-key` to replace `pre-init-spec`
 * Added `expand-key` and new `expand` function (#102)
+* Added babashka test runner (#86)
```
Comment (weavejester) [inline CHANGELOG.md:7]: Can you remove this line? I typically update the changelog when I have a release.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -30,6 +31,10 @@ jobs:
       - name: Run tests
         run: lein test-all
 
+      - name: Run bb tests
+        run: |
+          bb test:bb
```
Comment (weavejester) [inline .github/workflows/test.yml:36]: Maybe: `run: bb test:bb`, and we save a line.

**Comment by borkdude** [comment]
All processed, hopefully

**Comment by weavejester** [comment]
Perfect! Thanks!

**Pull request merged**